dist/index.js

```diff
@@ -22,8 +22,14 @@ const utils_1 = __webpack_require__(918);
 function run() {
     return __awaiter(this, void 0, void 0, function* () {
         try {
-            const requestReviewersResult = yield utils_1.requestReviewer();
-            core_1.setOutput('requestReviewersResult', requestReviewersResult);
+            const previouslyRequestedReviewers = yield utils_1.listRequestedReviewers();
+            if (previouslyRequestedReviewers.length < 1) {
+                // No reviewers yet, assign one
+                const requestedNewReviewer = yield utils_1.requestNewReviewer();
+                core_1.setOutput('requestedNewReviewer', requestedNewReviewer);
+            }
+            // Reviewer already assigned, do nothing
+            core_1.setOutput('previouslyRequestedReviewers', previouslyRequestedReviewers);
```

Comment:
question: did you mean to commit the build to the repo?

Reply:
yup, it can sound weird but github actions are read directly from the repo, and it's not able to read the TS/ES6+ code, so we actually need to commit the build/dist. Other option (that might be more elegant) is to have a branch for releases and on merge into master I trigger an action to build and commit the dist to that release branch instead... but this is for future releases.